go/cmake/golang.cmake

```diff
@@ -32,17 +32,21 @@ function(GO_LIBRARY NAME BUILD_TYPE)
   # will use the local changes in Paddle rather than checkout Paddle
   # in github.
   add_custom_target(copyPaddle
-    COMMAND ln -sf ${PADDLE_DIR} ${PADDLE_IN_GOPATH})
+    COMMAND rm -rf ${PADDLE_IN_GOPATH}/Paddle
```

Comment:
Strange, `-f` in `ln` command force the link, so we probably don't need to remove the old link.